### DISCUSSION TITLE
Changing Cron max_execution_time?

### DISCUSSION THREAD

**Discussion by AllOnMyOhm** [YetiForceCompany/YetiForceCRM#14134] [How to] [answered]
Hello folks, 

I'm new to this whole thing but this discussion thread has thus far been endlessly helpful, I was wondering if anyone could point me in the right direction as to how I might go about changing the max_execution_time for Cron?

**Comment by apcloic** [comment, marked as answer]
@AllOnMyOhm
Hi,
In config/Main.php

````	/** The maximum time of executing a cron. Recommended the same as the max_exacution_time parameter value. */
	public static $maxExecutionCronTime = 3600;
````

**Reply by AllOnMyOhm** [reply]
Thank you!
